doc/en/source/https.rst

```diff
@@ -262,28 +265,38 @@ Change the IP configuration accordingly
 
 Configuration file need to be modified at the following places with IP addresses  according to user own environment.
 
-- Change PEP Proxy host-port and container port in docker-compose.idm.yml file.
```

Comment:
should both port be the same? Could we use any port? Why not provide a default port/configuration for this?

Reply:
The line (line no 265) to which you are pointing is no longer part of manual.